package.json

```diff
@@ -18,7 +18,11 @@
     "sonar-scan": "sonar-scanner",
     "test:smoke": "export NODE_TLS_REJECT_UNAUTHORIZED=0 && jest -c jest.smoke.config.js",
     "test:functional": "export NODE_TLS_REJECT_UNAUTHORIZED=0 && jest -c jest.smoke.config.js",
-    "cichecks": "yarn && yarn build && yarn lint  && yarn test && yarn test:routes &&  yarn test:a11y"
+    "cichecks": "yarn && yarn build && yarn lint  && yarn test && yarn test:routes &&  yarn test:a11y",
+    "prepare": "husky install"
+  },
+  "lint-staged": {
```

Comment:
I don't think we should be overriding a default lint staged config. Because we're losing eslint check just now.

Reply:
Correct, removed now. This block caused lint-staged to run twice.